docs/learn-staking.md

```diff
@@ -137,10 +144,26 @@ GRANDPA and BABE equivocation slashing penalty is calculated as below:
 Validators may run their nodes on multiple machines to make sure they can still perform validation work in case one of their nodes goes down. It should be noted that if they do not have good coordination to manage signing machines, then equivocation is possible.
 
 > Notice:
-If a validator is reported for any one of the offences they will be removed from the validator set and they will not be paid while they are kicked out.
+If a validator is reported for any one of the offences they will be removed from the validator set ([chilled](#chilling)) and they will not be paid while they are kicked out.
 
 If you want to know more details about slashing, please look at our [research page](https://research.web3.foundation/en/latest/polkadot/slashing/amounts.html).
 
+### Chilling
+
+Chilling is the act of removing a validator from the active validator set, also disqualifying them from the set of electable candidates in the next NPoS cycle. This carries an implied penalty of being unable to earn rewards for the duration of the next era, and requires them to earn back any nominations they may have had before being removed.
+
+Because every offence regardless of level triggers a chill, and every chill triggers an automatic re-election of a new active validator set, the offending validator is immediately removed from the current set and the next, and someone else takes their place. To re-join, they need to acknowledge the slash and re-announce their candidature.
```

Comment:
I thought that this changed recently?  We had the issue where a huge group got chilled and all the nominators lost the ability to nominate them, causing a chain reaction which pushed down the amount of stake in the system.

Reply:
I have been unable to verify this yet. @AlfonsoCev can you shed some light perhaps?

Reply:
I recommend looking into these unit tests in `frame/staking`

- `offence_deselects_validator_even_when_slash_is_zero`
- `nomination_slash_filter_is_checked`
- `zero_slash_keeps_nominators`

and what I can add is: 
- If a validator is slashed, the validator is chilled, but the nominators are not. And there is an additional filter that checks if a validator has been slashed recently, then the nominator's who voted for the slashed validator  _before the slashing event_ are actually dropped (only the vote is dropped, not the entire nominator). 

I don't quite know when this recovers and when the validators can become effective again and re-gain the lost votes. @rphmeier will know more. 

```rust
targets.retain(|stash| {
	<Self as Store>::SlashingSpans::get(&stash).map_or(
		true,
		|spans| submitted_in >= spans.last_nonzero_slash(),
	)
});
```

And I know from experience that the easiest way to unravel such details from the code is to add new unit tests that check the hypothesis.

Reply:
Useful, thanks!

Reply:
I think we should clarify in the text that chilling and slashing are not necessarily linked. As Bill said, a validator can choose to chill if they need to stop their node for a while. In that case they continue being active in the current era and are inactive from the next era onward, and they don't lose any nominations. 

If I understand correctly, when there's an offense event chilling is just one of the consequences, being ignored for the remainder of the current era is another consequence, and losing nominations is another one (but this only for non-zero slashings).

Reply:
> I don't quite know when this recovers and when the validators can become effective again and re-gain the lost votes

At the moment, it does not. The nominators must re-issue their votes on those validators to re-activate.